<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recon_configuration_page_global_timezone</class>
 <widget class="QWidget" name="recon_configuration_page_global_timezone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>609</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Case Time Zone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_timezones">
           <property name="minimumSize">
            <size>
             <width>271</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>271</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_date_format">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Date Format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_date_format">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_timezone">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>INT</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>End</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>start_epoch</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>end_epoch</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time Zone</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_add_timezone">
           <property name="minimumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_remove_timezone">
           <property name="minimumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_edit_timezone">
           <property name="minimumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>298</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
